dev-guide/build.md

```diff
@@ -0,0 +1,29 @@
+# Build
+
+## Overview
+
+If you want to dive into TiDB project, change some codes and send us PR, you may follow this guide to build TiDB project.
+
+You can see [user build guide](../op-guide/build.md) to check platforms and prerequisites first. 
+
+## Install RocksDB
+
+RocksDB 4.9+ is required, you can install RocksDB shared library manually according to [INSTALL.md](https://github.com/facebook/rocksdb/blob/master/INSTALL.md). 
+
+The [build RocksDB script](../scripts/build_rocksdb.sh) can help you build and install RocksDB in system path.
```

Comment:
Combine the two paras to one: RocksDB 4.9+ is required. You can install the RocksDB shared library manually according to [INSTALL.md](https://github.com/facebook/rocksdb/blob/master/INSTALL.md) or use the [RocksDB build script](../scripts/build_rocksdb.sh) to build and install RocksDB in the system path.